### ISSUE TITLE
Bsuite: Decide whether or not to use logarithmic logging for csvs.

### ISSUE THREAD

**Issue by tcfuji** [exalearn/EXARL#214]
Problem: Due to the new update method for bsuite in exaRL, logging now prints the stats for all episodes in the csv files. Might only need the default subset previously defined in bsuite.

**Comment by Jodasue** [comment]
No... it shouldn't for when we want to merge the results from across nodes.

**Comment by tcfuji** [comment]
Solved in PR #217
